### DISCUSSION TITLE
Add a new device, question

### DISCUSSION THREAD

**Discussion by cnekola** [Koenkk/zigbee2mqtt#10575] [Devices]
Hi,
First, I'm new with with zigbee2mqtt.
I have bought a new devices (https://www.zigbee2mqtt.io/devices/ERS-10TZBVK-AA.html) which has recently been supported. Unfortunately not full complete therefore I want to write my own external converter file.

I have two questions:

1)
In journalctl (journalctl -u zigbee2mqtt.service -f) I see two different  outputs:
Jän 04 22:12:07 zigbee2mqtt npm[568211]: Zigbee2MQTT:debug 2022-01-04 22:12:07: No converter available for 'ERS-10TZBVK-AA' with cluster 'genOnOff' and type 'raw' and data '{"data":[1,99,253,0],"type":"Buffer"}'

If I press the button fast three times I can "switch" to:
Jän 04 22:14:01 zigbee2mqtt npm[568211]: Zigbee2MQTT:debug 2022-01-04 22:14:01: No converter available for 'ERS-10TZBVK-AA' with cluster 'genOnOff' and type 'commandToggle' and data '{}'

In both variants I miss functions.
In "raw and data": Press and Rotate
In "clear text": Double Press

I'm completely clueless!

2)
Is it possible to change the "output action name"
Jän 04 22:38:12 zigbee2mqtt npm[7837]: Zigbee2MQTT:debug 2022-01-04 22:38:12: Received Zigbee message from 'SmartKnob', type 'commandToggle', cluster 'genOnOff', data '{}' from endpoint 1 with groupID 0
Jän 04 22:38:12 zigbee2mqtt npm[7837]: Zigbee2MQTT:info  2022-01-04 22:38:12: MQTT publish: topic 'zigbee2mqtt/SmartKnob', payload '{"action":"toggle","linkquality":63}'

From "toggle" to "click" or whatever. I reused an existing converters.
My .js File:

const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
const exposes = require('zigbee-herdsman-converters/lib/exposes');
const reporting = require('zigbee-herdsman-converters/lib/reporting');
const extend = require('zigbee-herdsman-converters/lib/extend');
const e = exposes.presets;
const ea = exposes.access;
const tuya = require("zigbee-herdsman-converters/lib/tuya");

const definition = {
    fingerprint: [
        {
            modelID: 'TS004F',
            manufacturerName: '_TZ3000_4fjiwweb'
        },
    ],
    model: 'ERS-10TZBVK-AA',
    vendor: 'TuYa',
    description: 'Smart Knob - Light Switch Scene Controllers',
    fromZigbee: [
        fz.command_step,
        fz.command_toggle,
        fz.command_step_color_temperature,
        fz.color_colortemp,
        fz.command_move_hue,
    ],
    toZigbee: [],
    onEvent: tuya.onEventSetTime,
    configure: async (device, coordinatorEndpoint, logger) => {
        const endpoint = device.getEndpoint(1);
        await reporting.bind(endpoint, coordinatorEndpoint, ['genBasic']);
    },
    exposes: [
        e.action(['toggle', 'hue_move', 'brightness_stop_down', 'brightness_stop_up', 'color_temperature_step_down', 'brightness_stop_up'])
                .withDescription('Smart Knob action status'), 
    ],
};

module.exports = definition;

thx, Chris

**Comment by toogooda** [comment]
Sorry I can't help with above but can you please tell me how you managed to pair this device I have been trying for days..
Thanks

**Reply by cnekola** [reply]
Hi toogooda,

sorry I can't help.
paring was was no problem. just tested with a new device.
I press the reset button for few minutes. 

br, chris

**Reply by toogooda** [reply]
@cnekola thanks for the reply, I wonder if I have a faulty one then have tried with 2 coordinators nothing gets found, never have issues with pairing anything before.

**Reply by wildekek** [reply]
Same for me: https://github.com/Koenkk/zigbee2mqtt/issues/11149
